Assets/AnkrSDK/Examples/UseCases/WebGlLogin/WebGLConnectionController.cs

```diff
@@ -0,0 +1,48 @@
+using AnkrSDK.Data;
+using UnityEngine;
+
+namespace AnkrSDK.UseCases.WebGlLogin
+{
+	public class WebGLConnectionController : MonoBehaviour
+	{
+		[SerializeField]
+		private WebGl.WebGLConnect _webGlConnect;
+
+		[SerializeField]
+		private WebGLLoginPanelController _webGlLoginManager;
+		
+		[SerializeField]
+		private GameObject _sceneChooser;
+
+		private void Start()
+		{
+			_sceneChooser.SetActive(false);
+			
+			_webGlConnect.OnNeedPanel.AddListener(ActivatePanel);
+			_webGlConnect.OnConnect.AddListener(ChangeLoginPanel);
+			_webGlLoginManager.NetworkHasChosen.AddListener(SetNetworkWhenItChosen);
+			_webGlLoginManager.WalletHasChosen.AddListener(SetWalletWhenItChosen);
+		}
+
+		private void ActivatePanel()
+		{
+			_webGlLoginManager.ShowPanel();
+		}
+
+		private void ChangeLoginPanel(WebGL.WebGLWrapper provider)
+		{
+			_webGlLoginManager.HidePanel();
+			_sceneChooser.SetActive(true);
+		}
+
+		private void SetNetworkWhenItChosen(NetworkName network)
```

Comment:
OnNetworkChosen

Reply:
Is it ok to name methods without verb in heading?

Reply:
chosen is a verb

Reply:
the rule is simple: event is named as "SomethingHappened" and subscription is named "OnSomethingHappened"